great_ape_safe/ape_api/pancakeswap_v2.py

```diff
@@ -0,0 +1,15 @@
+from brownie import web3
+from great_ape_safe.ape_api.uni_v2 import UniV2
+from helpers.addresses import registry
+
+
+class Pancakeswap(UniV2):
+    def __init__(self, safe):
+        self.safe = safe
+
+        self.router = safe.contract(registry.bsc.pancakeswap.router_v2)
+        self.factory = safe.contract(registry.bsc.pancakeswap.factory_v2)
+
+        self.max_slippage = 0.02
+        self.max_weth_unwrap = 0.01
+        self.deadline = 60 * 60 * 12
```

Comment:
these will get inherited from UniV2 if left undeclared. so if using same values as univ2 these could be left out.

Reply:
i tried removing them, but while running got errors

```
  File "./great_ape_safe/ape_api/uni_v2.py", line 153, in swap_exact_tokens_for_eth
    amountOut * (1 - self.max_slippage),
AttributeError: 'PancakeswapV2' object has no attribute 'max_slippage'
```

Reply:
ah yea sorry my bad, that is because they are not part of the global class like the defined funcs, but only initialised through __init__. guess we could move them outside the __init__, but some other time :)